services/api/db/serializers.py

```diff
@@ -0,0 +1,101 @@
+from rest_framework import serializers
+from db.models import User
+from django.utils.encoding import smart_str, force_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.core.validators import validate_email
+
+#Method for validating password.
+def validate_confirm_password(password,confirm_password):
+    if password != confirm_password:
+        raise serializers.ValidationError("Password and confirm password don't match.")
```

Comment:
Instead of raising exception here, can we just return True or False. This way, the duty of this function is only to validate the logic and give the result. The error handling and other things can be done at the consumer end.

Reply:
yes applied this way of writing method